push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/TokenLoader.java

```diff
@@ -46,9 +53,6 @@
 @Stateless
 public class TokenLoader {
 
-    private final static int NUMBER_OF_BATCHES = 10;
-    private final static int BATCH_SIZE = 1000;
```

Comment:
These guys are still referenced on the JavaDoc: https://github.com/lfryc/aerogear-unifiedpush-server/blob/service-reuse-and-batch-size/push/sender/src/main/java/org/jboss/aerogear/unifiedpush/message/TokenLoader.java#L80-L88